frontend/src/libs/components/modal/styles.module.scss

```diff
@@ -0,0 +1,65 @@
+.overlay {
```

Comment:
Actually yes, by changing any property it would be somehow broken. At least I tried to make overlay, which should cover whole page, that isn't common dialog style behaviour